bin/gtf2bed.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+
+# MIT License
+#
+# Copyright (c) Tobias Neumann, 2019
```

Comment:
Did you write this script? We have a different version that does presumably the same thing in the rnaseq pipeline [here](https://github.com/nf-core/rnaseq/blob/master/bin/gtf2bed)..

Reply:
I think so, because I actually only extract the 3' UTRs from the gtf and put it into bed format.